src/services/affinidi/auth/token.ts

```diff
@@ -101,11 +111,11 @@ const storer: CredSetterGetter = {
   setProjectToken: (token: ProjectToken): void => {
     credentialConf.set('projectToken', token)
   },
-  getPrincipalId: (): string => {
-    return credentialConf.get('principalId')
+  setPrincipal: (principal: Principal): void => {
+    credentialConf.set('principal', principal)
```

Comment:
Let's rename credentialConf to credentialConfig

Reply:
done

Reply:
done